mmseg/models/segmentors/encoder_decoder.py

```diff
@@ -216,9 +216,13 @@ def whole_inference(self, img, img_meta, rescale):
 
         seg_logit = self.encode_decode(img, img_meta)
         if rescale:
+            if torch.onnx.is_in_onnx_export():
```

Comment:
We may add commets here. The change is for exporting to ONNX with dynamic shape.